lib/expansion_rules.rb

```diff
@@ -106,7 +106,7 @@ def details_fields(*fields)
 
   POSSIBLE_FIELDS_FOR_LINK_EXPANSION = DEFAULT_FIELDS +
     %i[details] +
-    %i[id state phase unpublishings.type] -
+    %i[id state phase description unpublishings.type] -
```

Comment:
tangent question: is there a particular reason why `:details` has it's own array? Just line length/readability?

Reply:
I'm not actually sure, but I wanted to keep it as it is. I think it probably is for readability.